Generation/src/components/MomentumRangeParticleGun.h

```diff
@@ -12,8 +12,8 @@
  *
  *  Particle gun with given momentum range
  *
- *  @author Patrick Robbe
- *  @author Benedikt Hegner
+ *  @author Patrick Robbe (adaptation to tool structure)
```

Comment:
No, this was the comment that was originally in the src file. I guess "tool structure" can mean Gaudi.